tests/src/test/scala/pureconfig/BasicConvertersSuite.scala

```diff
@@ -94,6 +94,8 @@ class BasicConvertersSuite extends BaseSuite {
     ConfigValueFactory.fromAnyRef("no") -> false,
     ConfigValueFactory.fromAnyRef("off") -> false)
 
+  checkArbitrary[Byte]
```

Comment:
Nitpick: Please move this down to around line 120 near where `Short` is checked, please.